numba_dppy/dppl_passes.py

```diff
@@ -38,6 +40,80 @@ def dpnp_available():
     except:
         return False
 
+rewrite_function_name_map = {"sum": (["np"], "sum"),
```

Comment:
The new pass should be in its own file. And please format with `black`.

Reply:
Moved to new file.